src/text-expander-element.ts

```diff
@@ -129,6 +129,8 @@ class TextExpander {
 
     this.input.value = beginning + value + remaining
 
+    this.input.dispatchEvent(new Event('change', {bubbles: true}))
```

Comment:
Can we please add a test that confirms this behaviour, that we can also use to validate and avoid regressions here.

Reply:
@keithamus , I don't have experience in testing. can you please help me out here?

Reply:
@keithamus, I have added some tests by looking at previous tests.